File: backend/src/main/java/cohappy/backend/model/dto/request/AddMessageDTO.java
package cohappy.backend.model.dto.request;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AddMessageDTO {
    private String chatCode;
    private String message;
    private byte[] messageImmage;
    private String userCode;
}

File: frontend/app/src/main/java/cohappy/frontend/model/dto/request/AddMessageDTO.kt
package cohappy.frontend.model.dto.request
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class AddMessageDTO(
    val chatCode: String,
    val message: String,
    var messageImmage: ByteArray? = null,
    val userCode: String
)
